python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -179,15 +179,16 @@ def __init__(
 
         Args:
             writer: The actor that may write to the channel. None signifies the driver.
-            readers: The actors that may read from the channel. None signifies
-                the driver.
+            readers: The actors that may read from the channel. No reader may be None.
             typ: Type information about the values passed through the channel.
                 Either an integer representing the max buffer size in bytes
                 allowed, or a SharedMemoryType.
         Returns:
             Channel: A wrapper around ray.ObjectRef.
         """
         assert len(readers) > 0
+        for reader in readers:
+            assert reader is not None
```

Comment:
Perhaps better to assert that they are all `isinstance(ActorHandle)`, which will also include this case.